pds_doi_core/input/osti_input_validator.py

```diff
@@ -0,0 +1,48 @@
+#!/bin/python
+#
+#  Copyright 2020, by the California Institute of Technology.  ALL RIGHTS
+#  RESERVED. United States Government Sponsorship acknowledged. Any commercial
+#  use must be negotiated with the Office of Technology Transfer at the
+#  California Institute of Technology.
+#
+# ------------------------------
+
+from lxml import etree
+from lxml import isoschematron
+
+from pds_doi_core.util.general_util import get_logger
+
+logger = get_logger('pds_doi_core.input.osti_input_util')
+
+class OSTIInputValidator:
+    # This OSTIInputValidator provides some functions to validate the input to 'release' action specific to OSTI.  Other data center may have different format.
+
+    def validate_release_input(self, input_to_osti):
```

Comment:
You can rename the method 'validate' since the rest is already said in the object name.

Reply:
I was thinking that perhaps when we validate the 'deactivate' DOI next, it will use different a schematron.  We will need to make a distinction in the code if the validation is either for 'release' or 'deactivate'.

Reply:
From what I am seeing in OSTI documentation (https://www.osti.gov/iad2/docs#examples-deactivate) deactivate operation will not require to submit a XML. So I don't think we will need to validate an input here.

But you are right we might want to add more case later. I would still keep just "validate". I will not be too late to rename that method when we actually realize we need different flavor of it (it is very easy to rename with PyCharm IDE).

Reply:
Function has been renamed per your suggestion.

Reply:
Ok, perfect